pkgs/applications/networking/instant-messengers/zoom-us/default.nix

```diff
@@ -71,21 +92,39 @@ stdenv.mkDerivation rec {
 
   src = srcs.${stdenv.hostPlatform.system};
 
-  dontUnpack = true;
+  dontUnpack = if stdenv.isLinux then true else null;
```

Comment:
```suggestion
  dontUnpack = stdenv.isLinux then true else false;
```

Reply:
I wouldn't remove the `if`, because it helps readability, but changing the `null` to `false` is a matter of taste IMO.

Reply:
> I wouldn't remove the `if`, because it helps readability,

Not sure why I removed that.